VMEncryption/main/oscrypto/rhel_72/encryptstates/PatchBootSystemState.py

```diff
@@ -75,9 +75,29 @@ def enter(self):
             self.command_executor.ExecuteInBash('for i in dev proc sys boot; do mount --move /oldroot/$i /$i; done')
 
             extension_full_name = 'Microsoft.Azure.Security.' + CommonVariables.extension_name
+            extension_versioned_name = 'Microsoft.Azure.Security.' + CommonVariables.extension_name + '-' + CommonVariables.extension_version
+            test_extension_full_name = CommonVariables.test_extension_publisher + CommonVariables.test_extension_name
+            test_extension_versioned_name = CommonVariables.test_extension_publisher + CommonVariables.test_extension_name + '-' + CommonVariables.extension_version
             self.command_executor.Execute('cp -ax' +
                                           ' /var/log/azure/{0}'.format(extension_full_name) +
                                           ' /oldroot/var/log/azure/{0}.Stripdown'.format(extension_full_name))
+            self.command_executor.ExecuteInBash('cp -ax' +
+                                          ' /var/lib/waagent/{0}/config/*.settings.rejected'.format(extension_versioned_name) +
+                                          ' /oldroot/var/lib/waagent/{0}/config'.format(extension_versioned_name))
+            self.command_executor.ExecuteInBash('cp -ax' +
+                                          ' /var/lib/waagent/{0}/status/*.status.rejected'.format(extension_versioned_name) +
+                                          ' /oldroot/var/lib/waagent/{0}/status'.format(extension_versioned_name))
+            self.command_executor.Execute('cp -ax' +
```

Comment:
remove or comment out test hooks?

Reply:
Reason explained in previous comments. Let me know if it is fine.

Reply:
I understand the reasoning now as per previous comments.  It would be preferred to suppress the test-only portion in production to the extent possible.

Reply:
Add suppress logging for test env